### PR TITLE
Check for null response body with null body status

### PR DESCRIPTION
**Pull request by trivikr**
This PR attempt to add test which can reproduce null response body with null body status:
* added in `Chromium 113.0.5672.63` in https://github.com/chromium/chromium/commit/58a2d915312acf7e768f6dcb339021a09c33cdd9
* reverted in `Chromium 113.0.5672.92` in https://github.com/chromium/chromium/commit/ad69498c5f5dd231992bb4cc26b1c1b5e1db1219

### CODE REVIEW

File: .github/workflows/chrome_v112.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          cache: "yarn"
+      - uses: browser-actions/setup-chromium@v1
+        with:
+          chrome-version: "112.0"
```
Comment (trivikr) [inline .github/workflows/chrome_v112.yml]: The GitHub action attempts to retrieve from:
```console
Acquiring 112.0 from https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F112.0%2Fchrome-linux.zip?alt=media
```

When exploring the [Linux_x64 folder](https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/), the version is `112001`

```suggestion
          chrome-version: "112001"
```

Comment (trivikr) [reply .github/workflows/chrome_v112.yml]: The numbers in Linux_x64 folder are commit positions, and not version numbers

File: .github/workflows/chrome_v112.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          cache: "yarn"
+      - uses: browser-actions/setup-chromium@v1
+        with:
+          chrome-version: "112001"
```
Comment (trivikr) [inline .github/workflows/chrome_v112.yml]: Use branch position for v112 from https://chromiumdash.appspot.com/releases?platform=Linux
```suggestion
          chrome-version: "1109224"
```

File: .github/workflows/chrome_v112.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          cache: "yarn"
+      - uses: browser-actions/setup-chromium@v1
+        with:
+          chrome-version: "1109224"
```
Comment (trivikr) [inline .github/workflows/chrome_v112.yml]: Uses closest for v112, as 1109220 artifact is missing
From https://chromiumdash.appspot.com/releases?platform=Linux

```suggestion
          # Closest available commit position to official v112 from From https://chromiumdash.appspot.com/releases?platform=Linux
          chrome-version: "1109220"
```
